index.js

```diff
@@ -45,11 +45,25 @@ function setupMethods (soljson){
 		return JSON.parse(result);
 	}
 
+        var link = function(bytecode, libraries) {
+                for (var libraryName in libraries) {
+                        var libLabel = '__' + libraryName + Array(39 - libraryName.length).join('_');
```

Comment:
Well, just tried it out:

```js
library NameTooLong12345678XX12345678XX12345678XX12345678XX {
    function a() returns (bool) {
        return true;
    }
}

contract A {
    function a() returns (bool) {
        return NameTooLong12345678XX12345678XX12345678XX12345678XX.a();
    }
}
```

The output is: `__NameTooLong12345678XX12345678XX12345__`

It truncates it.